plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -227,8 +235,8 @@ def updateFirmware(self, file_name):
     def firmwareUpdateFinished(self):
         return self._firmware_update_finished
 
-    def resetFirmwareUpdateFinished(self):
-        self._firmware_update_finished = False
+    def resetFirmwareUpdate(self, update_has_finished = False):
```

Comment:
Is this called from anywhere other than from _updateFirmware_completed above? If not, why is it a public method, and what is the use of the argument?